bounded_sum_known_n/BoundedSumKnownN.tex

```diff
@@ -0,0 +1,394 @@
+    
+    Since by assumption \texttt{relation(d\_in, d\_out) = True}, by the \texttt{MakeBoundedSumN} stability relation (as defined in line~\ref{line:rel} in the pseudocode), we have that \texttt{2 *  d\_out >= inf\_cast(d\_in, T) * (U-L)}. Therefore, by Lemma \ref{lemma:pseudo-to-arith}, we also have that $\dout \geq \din * (U-L)/2$.
+    
+    
+    % We show that, if we prove that 
+    % This is equivalent to the stability relation $\dout \geq \texttt{inf\_cast}(\din, T)\cdot (U-L)/2$, which is the one we will show in the proof, except that we avoid any possible integer rounding errors by multiplying by 2 instead of dividing. We will also drop the \texttt{inf\_cast} in the rest of the proof, given that it ensures the correct type of $\din$ but does not affect the bounds of the relation. 
+    
+    We now show that, if $\dout \geq \din * (\texttt{U}-\texttt{L})/2$, then \texttt{function(v)}, \texttt{function(w)} are $\dout$-close under \texttt{output\_metric}.
+
+    
+    % end of trying to rewrite the paragraph above this
+    
+    Moreover, note in the statement of theorem \ref{thrm:privacy-proof} that $\texttt{v}, \texttt{w}$ are assumed to be $\din$-close. By the definition of the symmetric distance metric in the proof definitions document linked in Section \ref{sec:versioned-docs}, this is equivalent to stating that $d_{Sym}(v, w) = |\MultiSet(v) \Delta \MultiSet(w)| \leq \din$. Further, applying the histogram notation,\footnote{Note that there is a bijection between multisets and histograms, which is why the proof can be carried out with either notion. For further details, please consult the proof definitions document in Section \ref{sec:versioned-docs}.} it follows that
+    \[
+        d_{Sym}(v, w) = \lVert h_{v} - h_{w}\rVert_1 = \sum_z |h_v(z) - h_w(z)| \leq \din.
+    \]
+    We now want to show that
+    \[
+        d_{Abs}(\function(v), \function(w)) \leq d_{Sym}(v, w) \cdot \dfrac{U-L}{2}.
+    \]
+    
+    % start of reorganization
+    
+    To show that $d_{Abs}(\function(v), \function(w)) \leq d_{Sym}(v, w) \cdot \frac{U-L}{2}$, we will use the three lemmas described in the section ``The path property of symmetric distance on sized domains" from Section 5.2 in the proof definitions document (linked in section \ref{sec:versioned-docs}). By Lemma 5.7 in the proof definitions document (linked in Section \ref{sec:versioned-docs}), which is applicable to \texttt{MakeBoundedSumN} because \texttt{input\_domain} is a a sized domain and \texttt{input\_metric} is symmetric distance, it suffices to show the following: For all vectors $x, y \in$ \texttt{input\_domain} such that $d_{Sym}(x, y) = 2$, it follows that 
+    \[
+    d_{Abs}(\texttt{function}(x), \texttt{function}(y)) \leq U-L.
+    \]
+    
+    By Lemma 5.6 from the proof definitions document (linked in Section \ref{sec:versioned-docs}), we know that vectors $x, y$ only differ on one element, given that, by assumption, $d_{Sym}(x, y) = 2$.
+    

```

Comment:
Suggest: remove line break, and change next two sentences to read: "Thus, if we write $x=(x_1,x_2,\ldots,x_n)$ and $y=(y_1,\ldots,y_n)$ as vectors, there is a coordinate $j$ such that we can change $x$ in (only) coordinate $j$ to obtain a vector $z$ such that $\MultiSet(z)=\MultiSet(y)$.   This construction of $z$ gives us the following...